10309.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--Copyright 2018 Hua Xiang IOT. All rights reserved.
+
+-->
+<LWM2M xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://www.openmobilealliance.org/tech/profiles/LWM2M-v1_1.xsd">
+  <Object ObjectType="MODefinition">
+    <Name>DeviceDataContainer</Name>
+    <Description1><![CDATA[DeviceDataContainer Object of HuaXiang Company.]]></Description1>
+    <ObjectID>10309</ObjectID>
+    <ObjectURN>urn:oma:lwm2m:x:10309</ObjectURN>
+    <LWM2MVersion>1.1</LWM2MVersion>
+    <ObjectVersion>1.0</ObjectVersion>
+    <MultipleInstances>Multiple</MultipleInstances>
+    <Mandatory>Optional</Mandatory>
+    <Resources>
+      <Item ID="1">
+        <Name>BToothstate</Name>
+        <Operations>RW</Operations>
+        <MultipleInstances>Multiple</MultipleInstances>
+        <Mandatory>Mandatory</Mandatory>
+        <Type>Integer</Type>

```

Comment:
Maybe boolean value is better here? Same comment applies to other state resources below

Reply:
That's true,boolean value is better.